turbopuffer/backend.py

```diff
@@ -94,7 +94,7 @@ def make_api_request(self,
                 performance['request_time'] = time.monotonic() - request_start
                 # print(f'Request time (HTTP {response.status_code}):', performance['request_time'])
 
-                if response.status_code > 500:
+                if response.status_code > 500 or response.status_code == 408 or response.status_code == 429:
```

Comment:
We should be retrying 502s etc., probably anything above 500?

Reply:
Yep that's what the `> 500` here does

Reply:
OK, so the other way around, we should retry 500s :)

Reply:
You're saying you want to retry `>= 500`, not just `> 500`?

Reply:
Yeah, don't you agree?

Reply:
Sure, sounds good to me. Will change.